app/about-blank.html

```diff
@@ -0,0 +1,11 @@
+<html>
```

Comment:
about:blank should be a blank html document
![screenshot 2016-02-11 16 18 02](https://cloud.githubusercontent.com/assets/831718/12990982/207b7eb6-d0db-11e5-9bba-1e6fadb2164a.png)